projects/filigran-website/components/icon-popup.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { FC, ReactElement, useState } from 'react';
+
+interface PopupProps {
+    icon: ReactElement | null;
+    iconName: string;
+    onClose: () => void;
+}
+
+const Popup: FC<PopupProps> = ({ icon, iconName, onClose }) => {
+    const [copySuccess, setCopySuccess] = useState(false);
+
+    const handleCopy = () => {
+        const statement = `import { ${iconName} } from '@filigran-ui';`;
+        navigator.clipboard.writeText(statement)
```

Comment:
could we use the toast component instead ?
https://filigranhq.github.io/filigran-ui/docs/components/toast
The implementation come from : 
https://ui.shadcn.com/docs/components/toast